extensions/BUILD.gn

```diff
@@ -8,8 +8,7 @@ import("//extensions/buildflags/buildflags.gni")
 if (is_android || is_ios) {
   source_set("common") {
     sources = [
-      "//extensions/common/constants.cc",
-
+      # "//extensions/common/constants.cc",
```

Comment:
shouldn't we add `//build:chromeos_buildflags` to deps instead?

Reply:
@goodov 
yes, we should, fixed at https://github.com/brave/brave-core/pull/28944/commits/f118fe2e7a003d03028a34a8f7a1b31b75ca06d2